test/plug/error_handler_test.exs

```diff
@@ -63,6 +87,19 @@ defmodule Plug.ErrorHandlerTest do
     end
 
     assert_received {:plug_conn, :sent}
-    assert {403, _headers, "Something went wrong"} = sent_resp(conn)
```

Comment:
It seems you are changing an existing test as part of this PR.

Reply:
This was because `Plug.Conn.WrapperError` doesn't have a `plug_status` and now the `Plug.Conn.WrapperError` is being passed to `handle_errors`